### ISSUE TITLE
Option to disable smooth pixel scaling and use nearest neighbor

### ISSUE THREAD

**Issue by OpenRift412** [chocolate-doom/chocolate-doom#1686]
While I do think the implementation of a light bilinear filter for giving the illusion of more even scaling at lower resolutions is tasteful, I think having the ability to manually disable this smoothing to use nearest-neighbor scaling should also be an option. 

This isn't so much a problem for me, as I have a Windows environment variable that disables bilinear resolution scaling for SDL, but I think for most users having a dedicated toggle in the Display>Advanced section of the setup would be good.

**Comment by chungy** [comment]
Does the integer_scaling feature provide the functionality you desire?

**Comment by OpenRift412** [comment]
> Does the integer_scaling feature provide the functionality you desire?

Not really, I prefer having the game take up the full screen height instead of being letterboxed.

**Comment by Clownacy** [comment]
You say that you think it should be an option, and that it would be good, but you never actually give any reasons why.

**Comment by OpenRift412** [comment]
> You say that you think it should be an option, and that it would be good, but you never actually give any reasons why.

Because the bilinear-sharp filter that Choco uses doesn't look very good, especially at resolutions where it's no longer necessary.
